docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -102,6 +102,10 @@ properties:
   - Hive metastore authentication type. Possible values are `NONE` or
     `KERBEROS`.
   - `NONE`
+* - `hive.metastore.catalog-name`
+  - Hive metastore custom catalog name. By default, the catalog name used by HiveMetastore is "hive".
```

Comment:
`By default, the catalog name used by HiveMetastore is "hive".` -> Note that the catalog name used by default by the HiveMetastore is "hive"`

Reply:
`If this property remains unset, the metastore client will retain the current behavior.` -> this is a bit misleading in a few months from now.

Reply:
What about ?

> Hive metastore catalog name. Hive metastore catalog	abstraction concept allows	
> different	systems	to	connect	to	different independent catalogs	in	the	metastore. 
> The default Hive metastore catalog name is `hive` and it is being accessed also when leaving
> this configuration property empty.

Reply:
slightly modified, looks ok?

```
The term "Hive metastore catalog name" refers to the abstraction concept within Hive, enabling various systems to connect to distinct, independent catalogs stored in the metastore. By default, the catalog name in Hive metastore is set to "hive." When this configuration property is left empty, it defaults to accessing the "hive" catalog.
```

Reply:
cc @mosabua  for help.

Reply:
may be if we need a bit more changes to the doc, we can take-up in the next pr cc @mosabua